features/specs/meerderjarig-verklaarde-minderjarige.feature

```diff
@@ -0,0 +1,126 @@
+      En 2 maanden geleden is 'Bert' erkend door 'Aart'
+      En 1 maand geleden is 'Gerda' meerderjarig verklaard
+      Als 'gezag' wordt gevraagd van 'Bert'
+      Dan is het gezag over 'Bert' eenhoofdig ouderlijk gezag met ouder 'Gerda'
+
+    Voorbeeld: Het kind is erkend voordat de moeder meerderjarig is geworden
+      Gegeven de 19 jaar geleden geboren persoon 'Gerda'
+      En de 21 jaar geleden geboren persoon 'Aart'
+      En de 2 jaar geleden geboren persoon 'Bert' met één ouder 'Gerda'
+      En 2 jaar geleden is 'Bert' erkend door 'Aart'
+      Als 'gezag' wordt gevraagd van 'Bert'
+      Dan is het gezag over 'Bert' eenhoofdig ouderlijk gezag met ouder 'Gerda'
+
+  Regel: Als een ouder erkend heeft nadat de moeder meerderjarig verklaard is of meerderjarig is geworden, dan is er gezamenlijk gezag voor de moeder en de erkenner
+
+    Voorbeeld: Het kind is erkend nadat de moeder meerderjarig is verklaard
+      Gegeven de 17 jaar geleden geboren persoon 'Gerda'
+      En de 19 jaar geleden geboren persoon 'Aart'
+      En de 3 maanden geleden geboren persoon 'Bert' met één ouder 'Gerda'
+      En 2 maanden geleden is 'Gerda' meerderjarig verklaard
+      En 1 maand geleden is 'Bert' erkend door 'Aart'
+      Als 'gezag' wordt gevraagd van 'Bert'
+      Dan is het gezag over 'Bert' gezamenlijk ouderlijk gezag met ouder 'Gerda' en ouder 'Aart'
+
+    Voorbeeld: Het kind is erkend nadat de moeder meerderjarig is geworden
+      Gegeven de 19 jaar geleden geboren persoon 'Gerda'
+      En de 21 jaar geleden geboren persoon 'Aart'
+      En de 2 jaar geleden geboren persoon 'Bert' met één ouder 'Gerda'
+      En 3 maanden geleden is 'Bert' erkend door 'Aart'
+      Als 'gezag' wordt gevraagd van 'Bert'
+      Dan is het gezag over 'Bert' gezamenlijk ouderlijk gezag met ouder 'Gerda' en ouder 'Aart'
+
+    Voorbeeld: Het kind is erkend nadat de moeder meerderjarig is verklaard en erkenner is minderjarig
+      Gegeven de 17 jaar geleden geboren persoon 'Gerda'
+      En de 17 jaar geleden geboren persoon 'Aart'
+      En de 3 maanden geleden geboren persoon 'Bert' met één ouder 'Gerda'
+      En 2 maanden geleden is 'Gerda' meerderjarig verklaard
+      En 1 maand geleden is 'Bert' erkend door 'Aart'
+      Als 'gezag' wordt gevraagd van 'Bert'
+      Dan is het gezag over 'Bert' eenhoofdig ouderlijk gezag met ouder 'Gerda'
+
+    Voorbeeld: Het kind is erkend nadat de moeder meerderjarig is verklaard en meerderjarig geworden erkenner was bij erkenning nog minderjarig
+      Gegeven de 17 jaar geleden geboren persoon 'Gerda'
+      En de 18 jaar geleden geboren persoon 'Aart'
+      En de 10 maanden geleden geboren persoon 'Bert' met één ouder 'Gerda'
+      En 9 maanden geleden is 'Gerda' meerderjarig verklaard
+      En 8 maanden geleden is 'Bert' erkend door 'Aart'
+      Als 'gezag' wordt gevraagd van 'Bert'
+      Dan is het gezag over 'Bert' gezamenlijk ouderlijk gezag met ouder 'Gerda' en ouder 'Aart'
```

Comment:
op het moment is erkenner minderjarig en ontstaat er dan geen gezamenlijk gezag. artikel 253d: erkenner krijgt niet automatisch gezag, maar mag er bij de rechter om vragen of gezamenlijk met de moeder aantekenen in het gezagsregister

Doen ze dit niet, dus is er geen indicatie gezag bij het kind, blijft alleen de moeder eenhoofdig gezag behouden.

als de erkenner op het moment van erkenning 18+ is, dan ontstaat op dat moment wel gezamenlijk gezag